metricbeat/module/kibana/stats/_meta/fields.yml

```diff
@@ -4,122 +4,106 @@
     Kibana stats and run-time metrics.
   release: beta
   fields:
-    - name: cluster_uuid
+    - name: uuid
       type: keyword
       description: >
-        UUID of the Elasticsearch cluster to which Kibana connects.
+        Kibana instance UUID
     - name: name
+      type: text
+      description: >
+        Kibana instance name
+    - name: index
       type: keyword
       description: >
-        Kibana instance name.
-    - name: uuid
+        Name of Kibana's internal index
+    - name: host
       type: keyword
       description: >
-        Kibana instance uuid.
-    - name: version.number
+        Kibana instance hostname
+    - name: transport_address
       type: keyword
       description: >
-        Kibana version number.
-    - name: status.overall.state
+        Kibana server's hostname and port
+    - name: version
       type: keyword
       description: >
-        Kibana overall state.
+        Kibana version
+    - name: snapshot
+      type: boolean
+      description: >
+        Whether the Kibana build is a snapshot build
+    - name: status
+      type: keyword
+      description: >
+        Kibana instance's health status
+    - name: cluster_uuid
```

Comment:
In the future this should go under `elasticsearch.cluster.id` (not needed now)

Reply:
I can easily make the change now itself. Unless there is a reason to hold off until later?

Reply:
Feel free to do it now.